### PR TITLE
Add Event WG Project to readme

### PR DESCRIPTION
**Pull request by MSNev**
see #1688

### CODE REVIEW

**Review by trask — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -157,12 +157,13 @@ End-User WG|Every other week on Thursday at 10:00 PT|[Google Doc](https://docs.g
...
 Community Demo Application|Every Monday at 8:30 PT|[Google Doc](https://docs.google.com/document/d/16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98/edit)|[Slack](https://cloud-native.slack.com/archives/C03B4CWV4DA)|
 Functions as a Service (FAAS) |Every other Tuesday at 12:00 PM PT |[Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc/edit#)|[Slack](https://cloud-native.slack.com/archives/C04HVBETC9Z)|
 OpenTelemetry Profiling WG | Every other week on Thursday at 08:00 AM PT |[Google Doc](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C03J794L0BV)|
 Security | Weekly on Wednesday at 08:30 AM PT |[Google Doc](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C05A85QC281)|
+Event WG|Every Friday at 10:00 AM PT|TBD Google Doc|[Slack](https://cloud-native.slack.com/archives/C062HUREGUV)|
```
Comment (jack-berg) [inline README.md:166]:
```suggestion
Event WG|Every Friday at 10:00 AM PT|[Google Doc](https://docs.google.com/document/d/1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A/edit#heading=h.1nz72gv5evo2)|[Slack](https://cloud-native.slack.com/archives/C062HUREGUV)|
```

**Review by jack-berg — APPROVED** [review]
I'm coordinating with @arminru to create the calendar invite.

File: README.md (hunk 1 above)
Comment (arminru) [reply README.md:166]:
```suggestion
Event WG|Every Friday at 10:00 AM PT|[Google Doc](https://docs.google.com/document/d/1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A)|[Slack](https://cloud-native.slack.com/archives/C062HUREGUV)|
```

**Review by arminru — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (arminru) [reply README.md:166]: @jack-berg please make sure to pass on ownership to the document to the GC Google account as described here: https://github.com/open-telemetry/community/issues/1147#issuecomment-1234759420

**Comment by jack-berg** [comment]
FYI, an entry for this meeting has been added to the OpenTelemetry calendar.

File: README.md (hunk 1 above)
Comment (jack-berg) [reply README.md:166]: Done, thanks!

Comment (trask) [reply README.md:166]: (accepted ownership)

File: README.md [hunk 2]
```diff
@@ -157,12 +157,13 @@ End-User WG|Every other week on Thursday at 10:00 PT|[Google Doc](https://docs.g
...
 Community Demo Application|Every Monday at 8:30 PT|[Google Doc](https://docs.google.com/document/d/16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98/edit)|[Slack](https://cloud-native.slack.com/archives/C03B4CWV4DA)|
 Functions as a Service (FAAS) |Every other Tuesday at 12:00 PM PT |[Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc/edit#)|[Slack](https://cloud-native.slack.com/archives/C04HVBETC9Z)|
 OpenTelemetry Profiling WG | Every other week on Thursday at 08:00 AM PT |[Google Doc](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C03J794L0BV)|
 Security | Weekly on Wednesday at 08:30 AM PT |[Google Doc](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C05A85QC281)|
+Event WG|Every Friday at 10:00 AM PT|[Google Doc](https://docs.google.com/document/d/1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A/edit#heading=h.1nz72gv5evo2)|[Slack](https://cloud-native.slack.com/archives/C062HUREGUV)|
```
Comment (trask) [inline README.md:166]:
```suggestion
Event WG|Every Friday at 10:00 AM PT|[Google Doc](https://docs.google.com/document/d/1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A/edit)|[Slack](https://cloud-native.slack.com/archives/C062HUREGUV)|
```

**Pull request merged**